### PR TITLE
Document bundler-workarounds.rb

### PR DESCRIPTION
**Pull request by pitr-ch**
related to #17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,12 @@ It is possible to run today on an unmodified JDK 9 EA build, but at the moment
 this requires building Graal yourself and we don't recommend end-users try it.
 It will be supported when Java 9 is released.
 
+#### Installing gems
+
+TruffleRube cannot install gems out of the box yet, however there are
```
Comment (chrisseaton) [inline README.md]: 'TruffleRube' sounds cool!

**Comment by chrisseaton** [comment]
More documentation is great!

File: README.md (hunk 1 above)
Comment (pitr-ch) [inline README.md]: how did this happen? 'e' is not even close! :) Fixed

File: doc/user/installing-gems.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Installing gems
+
+As mentioned in the 
+[`README`](https://github.com/graalvm/truffleruby/tree/truffle-head/README.md) 
+TruffleRuby currently cannot run `gem install` out of the box because of incomplete
+support for openssl and Nokogiri. However there is a workaround which can be used
+to get both `gem install` and `bundler install` working.
+ 
+    ruby -r bundler-workarounds.rb -S gem install
```
Comment (eregon) [inline doc/user/installing-gems.md]: We don't need a `.rb` at the end, right?

File: doc/user/installing-gems.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    ruby -r bundler-workarounds.rb -S bundle install
+    ruby -r bundler-workarounds.rb -S bundle update
+    
+`bundle exec` does not need the `bundler-workarounds.rb` loaded. 
+    
+    ruby -S bundle exec bin/rails server
```
Comment (eregon) [inline doc/user/installing-gems.md]: Is bundler pre-installed? If not, how to install it?

Comment (pitr-ch) [inline doc/user/installing-gems.md]: gem install with workaround, I'll add it to the example above while removing extra .rb

**Comment by pitr-ch** [comment]
fixed in 5e12f4ae8441f95c6e2238573ae53df97a23ae12

**Pull request merged**